server/athenian/api/preloading/cache.py

```diff
@@ -239,35 +259,41 @@ def _done(self, *_):
 
 def get_memory_cache_options() -> Dict[str, Dict]:
     """Return the options for the MemoryCache."""
-    # TODO
-    return {}
-    # Example configuration:
-    # from athenian.api.models.metadata.github import PullRequest
-
-    # return {
-    #     "mdb": {
-    #         "prs": {
-    #             "cols": [
-    #                 PullRequest.acc_id,
-    #                 PullRequest.node_id,
-    #                 PullRequest.closed,
-    #                 PullRequest.closed_at,
-    #                 PullRequest.created_at,
-    #                 PullRequest.merged,
-    #                 PullRequest.merged_at,
-    #                 PullRequest.merged_by_login,
-    #                 PullRequest.repository_full_name,
-    #                 PullRequest.updated_at,
-    #                 PullRequest.user_login,
-    #                 PullRequest.hidden,
-    #             ],
-    #             "categorical_cols": [
-    #                 PullRequest.acc_id,
-    #                 PullRequest.merged_by_login,
-    #                 PullRequest.repository_full_name,
-    #                 PullRequest.user_login,
-    #             ],
-    #             "identifier_cols": [PullRequest.node_id],
-    #         },
-    #     },
-    # }
+    return {
+        "mdb": {
+            "prs": {
+                "cols": [
+                    PullRequest.acc_id,
+                    PullRequest.node_id,
+                    PullRequest.number,
+                    PullRequest.closed,
+                    PullRequest.closed_at,
+                    PullRequest.created_at,
+                    PullRequest.merged,
+                    PullRequest.merge_commit_id,
+                    PullRequest.merge_commit_sha,
+                    PullRequest.merged_at,
+                    PullRequest.merged_by_login,
+                    PullRequest.repository_full_name,
+                    PullRequest.updated_at,
+                    PullRequest.user_login,
+                    PullRequest.hidden,
+                    PullRequest.additions,
+                    PullRequest.deletions,
+                    PullRequest.htmlurl,
+                ],
+                "categorical_cols": [
+                    PullRequest.acc_id,
+                    PullRequest.merged_by_login,
+                    PullRequest.repository_full_name,
+                    PullRequest.user_login,
+                ],
+                "identifier_cols": [
+                    PullRequest.node_id,
+                    PullRequest.merge_commit_id,
+                    PullRequest.merge_commit_sha,
+                    PullRequest.htmlurl,
+                ],
+            },
+        },
+    }
```

Comment:
Some more columns were required, we'll be able to check memory once we expose it through prometheus. I didn't manually check it since my prod dump has an outdated schema and will take me a lot to check again.

Reply:
Well actually I can easily do it by preloading directly from production without dumping, I'll add it in a separate PR a script to do it that would be useful for development.